### DISCUSSION TITLE
[Node Operator Question] Using `op-reth`, sync is very slow, repeated warning: "updated forkchoice, but node is syncing"

### DISCUSSION THREAD

**Discussion by Tudmotu** [ethereum-optimism/developers#75] [Node Operators]
### Issue Description

Following [a question I asked in discord](https://discord.com/channels/667044843901681675/1080867873997729874/1174754463144226946), opening this discussion at the request of @sbvegan.

I am running a `op-reth`/`op-node` setup.

My setup seems to work and does sync, but very slowly (~2.5 blocks per second).

The following warning appears repeatedly in the logs:
```
lvl=warn msg="Derivation process temporary error" attempts=1 err="engine stage failed: temp: temporarily cannot insert new safe block: updated forkchoice, but node is syncing"
```
Here is [a larger sample from the logs](https://gist.github.com/Tudmotu/f1f6650ae325cb95f001399434cba6e1), in case that helps.

To make it easy to reproduce I've created a repo with my setup:
https://github.com/Tudmotu/base-reth

The setup should be simple but takes some time as it requires compiling Reth with the `optimism` feature.
Follow the instructions in the readme, which are basically:
1. Clone
2. Add your L1 RPC to `.env`
3. Add the `jwt.hex` file in the appropriate place
4. Run `docker-compose`

Past the initial warm up you will see it starts syncing and the warning appears every couple of seconds.

The `.env` file inside the repo contains all the `OP_*` env vars.

If anyone has any idea how can I improve the sync speed I would highly appreciate it :pray: 
Thanks!

**Comment by sbvegan** [comment]
Hey @Tudmotu, I spoke to @clabby this morning. He mentioned that y'all chatted yesterday and the information he gave you is the latest information. It sounds like this behavior is expected and it has something to do with using a remote rpc endpoint. Please feel free to share more information that you've found out.

At the moment OPLabs doesn't have the bandwidth to support `op-reth` in an official capacity. I hope that we're able to scale so that can change in the future. Or maybe some external contributors can start joining in on the support.

**Reply by sbvegan** [reply]
I'm going to close this thread for now. Let me know if you want to revisit it.
